src/main/java/org/scijava/log/Utils.java

```diff
@@ -0,0 +1,33 @@
+package org.scijava.log;
+
+import java.util.Iterator;
+
+/**
+ * @author Matthias Arzt
+ */
+
+@IgnoreAsCallingClass
+public class Utils {
```

Comment:
Let's please not have a utility class called `Utils`. We can name it more explicitly relating to what sort of utility method(s) it provides. How about... `StackTraces` or `Stacks` or `Threads`? There are some similar things already: `DebugUtils.getMainClassName()`, `DebugUtils.getStackDump()`, and `DebugUtils.getStackTrace(Throwable)`, all of which could go into the same utility class as this new method.